agenda/presentation/src/main/java/com/gkp/agenda/presentation/task/edittask/EditTaskScreen.kt

```diff
@@ -0,0 +1,146 @@
+package com.gkp.agenda.presentation.task.edittask
+
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.Close
+import androidx.compose.material3.HorizontalDivider
+import androidx.compose.material3.Icon
+import androidx.compose.material3.IconButton
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.material3.TextButton
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.gkp.agenda.presentation.R
+import com.gkp.agenda.presentation.compoments.AgendaBackground
+import com.gkp.agenda.presentation.compoments.AgendaItemDateTime
+import com.gkp.agenda.presentation.compoments.AgendaItemDescription
+import com.gkp.agenda.presentation.compoments.AgendaItemHeader
+import com.gkp.agenda.presentation.compoments.AgendaItemReminderTime
+import com.gkp.agenda.presentation.compoments.AgendaItemTitle
+import com.gkp.core.designsystem.theme.TaskyTextFieldColor
+import com.gkp.core.designsystem.theme.TaskyTheme
+import org.koin.androidx.compose.koinViewModel
+
+@Composable
+fun EditTaskScreen(
+    modifier: Modifier = Modifier,
+    onClickTaskTitle: () -> Unit = {},
+    onClickTaskDescription: () -> Unit = {},
+    onClickEditButton: () -> Unit = {},
+    onClickDeleteButton: () -> Unit = {},
+    onClickCloseButton: () -> Unit = {}
+) {
+    val viewModel = koinViewModel<EditTaskViewModel>()
+    EditTaskScreen(
+        onClickTaskTitle = onClickTaskTitle,
+        onClickTaskDescription = onClickTaskDescription,
+        onClickEditButton = {},
+        onClickDeleteButton = {},
+        onClickCloseButton = {}
+    )
+}
+
+@Composable
+private fun EditTaskScreen(
+    onClickTaskTitle: () -> Unit = {},
+    onClickTaskDescription: () -> Unit = {},
+    onClickCloseButton: () -> Unit = {},
+    onClickEditButton: () -> Unit = {},
+    onClickDeleteButton: () -> Unit = {}
+) {
+    AgendaBackground(
+        title = stringResource(R.string.edit_task),
+        navigationIcon = {
+            IconButton(onClick = { onClickCloseButton() }) {
```

Comment:
Hint, when the signature of your lambda matches the parameter, then you can assign it in a more concise way:

```
IconButton(onClick = onClickCloseButton)
```